lib/jwt_sessions/store_adapters/abstract_store_adapter.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+require 'redis'
```

Comment:
The purpose of abstractizing store adapters is to get rid of required deps.

So, we should not depend on `redis`. It becomes optional.

We should only _require_ it if Redis adapter is used. It's up to user to make sure that `redis` gem is available.

We can make [this trick](https://github.com/rails/rails/blob/master/actioncable/lib/action_cable/subscription_adapter/redis.rb#L5) from Rails to show a meaningful error to user if `redis` gem couldn't be found.

Reply:
cool!